api/Api.php

```diff
@@ -0,0 +1,154 @@
+<?php
+/**
+ * This file is part of the ArcTouch CodeChallenge requested during the Interview Process.
+ *
+ * @package CodeChallenge
+ * @author Matheus Muller Borges
+ * @version 0.0.1
+ */
+namespace CodeChallenge;
+
+require_once dirname(__DIR__).'/bootstrap.php';
+
+
+/**
+ * Class Api
+ * @package CodeChallenge
+ */
+class Api {
+	/** @var  Router */
+	private $router;
+
+	/**
+	 * @var \Tmdb\Client
+	 */
+	private $client;
+
+	/**
+	 * Holds the TMDb images base url from the TMDB Configuration API.
+	 * @var string
+	 */
+	private $imagesBaseUrl;
+
+
+	public function __construct() {
+		$token = new \Tmdb\ApiToken('1f54bd990f1cdfb230adb312546d765d');
```

Comment:
How would you be able to move this out of the repository?

Reply:
The Token could be defined through a constant in a config file. This file would be "git ignored" and a config sample file would be provided. The user would edit the file with its Token and rename it. Another approach is to obtain the token through a constructor method parameter. The user would edit the `api/index.php` and insert its Token.

Particularly, I prefer the first approach, as it'll keep all user configurations in a single file.